packages/client/src/rpc/modules/eth.ts

```diff
@@ -784,7 +786,16 @@ export class Eth {
     if (account === undefined) {
       return '0x0'
```

Comment:
Was about to say that this should include pending txpool msgs from this account, but this is impossible since accounts do need to exist because otherwise they cannot pay the min basefee of 7 wei.